src/msg/msg_types.h

```diff
@@ -31,6 +31,7 @@ class entity_name_t {
 public:
   __u8 _type;
   int64_t _num;
+  int _ntype;	// used by connect bind
```

Comment:
I suppose we don't add a field here since _ntype isn't necessary for transmitting.